Simplenote/Classes/SPNoteEditorViewController+Extensions.swift

```diff
@@ -191,6 +191,98 @@ extension SPNoteEditorViewController {
 }
 
 
+// MARK: - History
+//
+extension SPNoteEditorViewController {
+
+    /// Indicates if note history is shown on screen
+    ///
+    @objc
+    var isShowingHistory: Bool {
+        return historyViewController != nil
+    }
+
+    /// Shows note history
+    ///
+    @objc
+    func showHistory() {
```

Comment:
Please take a peek at how `presentOptionsController` or `presentShareController` work. Might be worth shuffling a couple things around (So that we take the `note` as a parameter, and get rid of the IUO usage).

Reply:
I actually think passing `note` as a parameter is confusing in this scenario. This code is inside the context of `SPNoteEditorViewController` and looking at this code I would assume that it should always work with the current note. If I see that a `note` is passed as a param I think that maybe it is a different note? But why note editor shows one note and presents options / share / history of another note? 🤔